open_rarity/rarity_ranker.py

```diff
@@ -0,0 +1,53 @@
+import math
+
+
+class RarityRanker:
+    """This class is used to rank a set of tokens given their rarity scores."""
+
+    @staticmethod
+    def rank_tokens(*, token_id_to_scores: dict[str, float]) -> dict[str, int]:
+        """Ranks a set of tokens, identified uniquely by a string, based on
+        the scores provided. The ranking is done assuming @token_id_to_scores
+        contains all tokens for a given cohort/collection to rank tokens for.
+        Scores that are higher indicate a higher rarity, and thus a lower rank.
+
+        Tokens with the same score will be assigned the same rank.
+        Example: 1, 2, 2, 2, 5.
+        Scores are considered the same if they are within about 9 decimal digits
+        of each other.
```

Comment:
Maybe clarify that this is the same as `RANK` as opposed to `DENSE RANK`?

Reply:
done